app/javascript/orangelight/hathi_connector.es6

```diff
@@ -10,9 +10,15 @@ export default class HathiConnector {
 
   process_hathi_data(hathiData) {
     const base_url = "https://babel.hathitrust.org/Shibboleth.sso/Login?entityID=https://idp.princeton.edu/idp/shibboleth&target=https%3A%2F%2Fbabel.hathitrust.org%2Fcgi%2Fpt%3Fid%3D"
-    let mdp_id = hathiData.items[0].htid
-    let url = base_url + mdp_id
-    insert_online_link(url, "hathi_link")
+    try {
```

Comment:
@hackmastera  I think the try needs to be around line 7 . I would expect the getJSON to be the thing to throw an exception.